weave/tests/test_validate.py

```diff
@@ -481,18 +487,21 @@ def test_validate_manifest_schema_additional_field(test_validate):
         file_system=test_validate.file_system
     )
 
-    warn_info = validate.validate_pantry(pantry)
-
     # Check that there is only one warning raised
-    assert len(warn_info) == 1
+    warning_list = validate.validate_pantry(pantry)
+    assert len(warning_list) == 1
+
+    # Sort the warnings so they are in proper order
+    warning_list = sorted(warning_list, key=lambda x: x.args[1])
```

Comment:
Why do we sort the warning_list here when we know that the list only contains 1 element?

Reply:
I did this mainly for future proofing.
Some tests were failing in certain environments due to the tests making assumptions about the position of warnings in the list.
Adding the sort will ensure future changes to the tests (ie more warnings are added and expected to be raised) will not forget to sort or check explicitly for the warnings in the list.